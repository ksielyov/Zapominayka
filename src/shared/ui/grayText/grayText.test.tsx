import renderer from 'react-test-renderer';
import {GrayText} from '@ui';
import {render} from '@testing-library/react-native';
import React from 'react';

test('text should render correctly', () => {
  const tree = renderer.create(<GrayText>test</GrayText>).toJSON();
  expect(tree).toMatchSnapshot();
});

test('rendered text should be equal to `testText`', () => {
  const testText = 'nestedView';

  const {getByText} = render(<GrayText>{testText}</GrayText>);
  const element = getByText(testText);
  expect(element).toBeTruthy();
});
