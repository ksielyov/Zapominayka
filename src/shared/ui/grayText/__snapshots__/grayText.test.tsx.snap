// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`text should render correctly 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "#fff",
        "fontFamily": "Roboto-Regular",
        "fontWeight": "400",
        "opacity": 0.5,
      },
      Object {
        "fontSize": 29.426470588235293,
        "marginBottom": undefined,
        "marginTop": undefined,
      },
    ]
  }
>
  test
</Text>
`;
